### PR TITLE
Spacies Effect/Misc Cleanup

### PR DESCRIPTION
**Pull request by O1yyy**
WOLF
- Shine effects cleaned up, visually clearer when shine comes out

https://user-images.githubusercontent.com/19886440/224582048-24eee7b2-9853-4b21-90d8-d2aa5b43add4.mp4

- Neutral B effects/sounds lined up to match projectile frame

- Down Tilt nerfs were missing on last Wolf PR, reimplemented here: 
   - Body hitbox: Size: 5.0 -> 4.0 
   - Leg hitbox: Size: 4.0 -> 3.5 
   - Foot hitbox: Size: 5.0 -> 3.5 Angle: 75 -> 80

image

FOX
- Shine effects cleaned up, visually clearer when shine comes out

https://user-images.githubusercontent.com/19886440/224547301-dca6d832-a00a-4393-9779-a7d8ef914a3a.mp4

FALCO

- Shine effects cleaned up, no longer jittery and more visually clearer

https://user-images.githubusercontent.com/19886440/224547494-c89d2f2a-f117-4d50-a0ae-2811f86a5fa4.mp4

- Shine hitbox
  - Moved -2.0 units behind
 - Shine effects moved -2.0 to match
 
dotnet_2023-03-12-17-01-16

-------------------------------------------------------------------------------------------------------------------------------

General notes:
> I also discovered all 3 shines... reflect on the same frame as their hitbox, be it Fox/Falco F1 or Wolf's F3, and I wanted to mention that shine in past games only starts reflecting after F5. So while considering implementing that in this PR, I couldn't figure out where Falco's actual reflecting hitbox was being declared, so I gave up on changing any of them pertaining this behavior. I can revisit the idea if anyone can help me locate Falco's ReflectModule call.

### CODE REVIEW

**Review by HisashiASM — COMMENTED** [review]
LGTM, Quick and Easy.

File: fighters/fox/src/acmd/specials.rs [hunk 1]
```diff
@@ -100,40 +100,74 @@ unsafe fn sound_specialhi(fighter: &mut L2CAgentBase) {
 unsafe fn game_speciallwstart(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
-    frame(lua_state, 1.0);
+    frame(lua_state, 0.0);
```
Comment (WuBoytH) [inline fighters/fox/src/acmd/specials.rs]: Does a motion rate on frame 0 produce different results than putting it on frame 1? Test this and get back to me

**Review by WuBoytH — CHANGES_REQUESTED** [review]

File: fighters/fox/src/acmd/specials.rs [hunk 2]
```diff
@@ -100,40 +100,74 @@ unsafe fn sound_specialhi(fighter: &mut L2CAgentBase) {
...
     if is_excute(fighter) {
+        FT_MOTION_RATE(fighter, 2.0);
```
Comment (WuBoytH) [inline fighters/fox/src/acmd/specials.rs]: once again people are putting `FT_MOTION_RATE` in `is_excute`, don't do that

Comment (SuddyN) [reply fighters/fox/src/acmd/specials.rs]: oops, why?

**Comment by jobrien97** [comment]
If the only point of motion rating this was to clean up the GFX, please use `LAST_EFFECT_SET_RATE` instead, in the effect script

**Comment by O1yyy** [comment]
> If the only point of motion rating this was to clean up the GFX, please use `LAST_EFFECT_SET_RATE` instead, in the effect script

To elaborate on Fox's MotionRate call, there was strange behaviour with the OPFF condition where changing the number below 2.0 would cause him to JC on frame 3, and >2.0 would jc on frame 5, i didnt know how to go about it exactly cuz the numbers didnt make sense, so i tried simply virtually skipping a frame while it was set to 2.0 (jc F5), and getting to "frame 5" a frame earlier, essentially reaching F4 that way

I was going to consult you guys on why the OPFF condition is even behaving that way, if you guys have a better solution please be my guest i would like to for these numbers to just make sense lol

File: fighters/fox/src/acmd/specials.rs (hunk 1 above)
Comment (O1yyy) [reply fighters/fox/src/acmd/specials.rs]: Tested this, yeah didn't need to be on F0, behaves the same with the call on F1
Please refer to my logic in the latest comment though, I'd like to know if i even need to do this at all

**Comment by WuBoytH** [comment]
> > If the only point of motion rating this was to clean up the GFX, please use `LAST_EFFECT_SET_RATE` instead, in the effect script
> 
> To elaborate on Fox's MotionRate call, there was strange behaviour with the OPFF condition where changing the number below 2.0 would cause him to JC on frame 3, and >2.0 would jc on frame 5, i didnt know how to go about it exactly cuz the numbers didnt make sense, so i tried simply virtually skipping a frame while it was set to 2.0 (jc F5), and getting to "frame 5" a frame earlier, essentially reaching F4 that way
> 
> I was going to consult you guys on why the OPFF condition is even behaving that way, if you guys have a better solution please be my guest i would like to for these numbers to just make sense lol

You're using motion frame to determine when he should be able to jump cancel
That of course will be affected by any alterations to the motion's rate.
If you want this to be unaffected by this you want to check status frame instead, which I'm unsure if there's a fighter extension for.

**Comment by O1yyy** [comment]
> > > If the only point of motion rating this was to clean up the GFX, please use `LAST_EFFECT_SET_RATE` instead, in the effect script
> > 
> > 
> > To elaborate on Fox's MotionRate call, there was strange behaviour with the OPFF condition where changing the number below 2.0 would cause him to JC on frame 3, and >2.0 would jc on frame 5, i didnt know how to go about it exactly cuz the numbers didnt make sense, so i tried simply virtually skipping a frame while it was set to 2.0 (jc F5), and getting to "frame 5" a frame earlier, essentially reaching F4 that way
> > I was going to consult you guys on why the OPFF condition is even behaving that way, if you guys have a better solution please be my guest i would like to for these numbers to just make sense lol
> 
> You're using motion frame to determine when he should be able to jump cancel That of course will be affected by any alterations to the motion's rate. If you want this to be unaffected by this you want to check status frame instead, which I'm unsure if there's a fighter extension for.

Alright just tried it with fighter.status_frame() with the motion rate calls disabled; 
\>= 2 causes F3, 
\>2 causes F5

ima need u to slap its hand or somn or else my motion rate solution fixes this

**Review by WuBoytH — APPROVED** [review]
hmm.
spacies.
yes.

**Pull request merged**